test/SlowTests/MailingList/CanTransformWithDynamicFields3.cs

```diff
@@ -1,223 +0,0 @@
-using System.Collections.Generic;
-using System.Linq;
-using FastTests;
-using Newtonsoft.Json.Linq;
-using Raven.Client;
-using Raven.Client.Documents;
-using Raven.Client.Documents.Indexes;
-using Raven.Client.Documents.Transformers;
-using SlowTests.Utils;
-using Xunit;
-
-namespace SlowTests.MailingList
-{
-    public class CanTransformWithDynamicFields : RavenTestBase
```

Comment:
Restore candidate: I think this one is worth to be restored since it deals with dynamic fields